vms/avm/config/config.go

```diff
@@ -3,11 +3,23 @@
 
 package config
 
+import "time"
+
 // Struct collecting all the foundational parameters of the AVM
 type Config struct {
 	// Fee that is burned by every non-asset creating transaction
 	TxFee uint64
 
 	// Fee that must be burned by every asset creating transaction
 	CreateAssetTxFee uint64
+
+	// Time of the Durango network upgrade
+	DurangoTime time.Time
```

Comment:
Why are we adding `DurangoTime`?

Reply:
I have messed up the merge I think. Thanks for spotting this, removed.